tests/main/interfaces-account-control/task.yaml

```diff
@@ -27,11 +27,8 @@ restore: |
     snap remove "$TSNAP"
 
 execute: |
-    #shellcheck source=tests/lib/dirs.sh
-    . "$TESTSLIB"/dirs.sh
-
-    "$SNAP_MOUNT_DIR"/bin/"$TSNAP".useradd --extrausers alice
-    echo alice:password | "$SNAP_MOUNT_DIR"/bin/"$TSNAP".chpasswd
+    snap run "$TSNAP".useradd --extrausers alice
```

Comment:
Silly question - why do we have the "snap run" here at all? Could we simply use `"$TSNAP".useradd` here?